web/_layouts/schema.html

```diff
@@ -60,6 +60,7 @@ <h2>{{ slug }} <small><span class="label">{{ page.version }}</span></small></h2>
       </div>
       {% endif %}
       {{ content }}
+      {% include consolidation.html schema_data=schema_data %}
```

Comment:
The content of the page is the schema's README + the info block at the very beginning. See [a live page](http://localhost:4000/etalab/schema-stationnement/latest.html) as a reminder